package.json

```
@@ -16,7 +16,8 @@
     "lint:fix": "next lint --fix",
     "preview": "next build && next start",
     "start": "next start",
-    "typecheck": "tsc --noEmit"
+    "typecheck": "tsc --noEmit",
+    "scan": "next dev & npx react-scan http://localhost:3000"
```

```suggestion
    "scan": "pnpm dev & npx react-scan http://localhost:3000"
```

we can reuse `dev` script which uses turbopack